seed_isort_config.py

```diff
@@ -138,6 +138,7 @@ def main(argv: Optional[Sequence[str]] = None) -> int:
             else:
                 with open(filename, 'w', encoding='UTF-8', newline='') as f:
                     f.write(new_contents)
+                print(f'\'{os.path.basename(filename)}\' updated.')
```

Comment:
- please add a test demonstrating the change
- no need to put a filename in quotes, but if you were going to do that you have two better options:
    - use double quotes to avoid the escape sequences
    - use `!r` so it prints the string's representation

also I'd just use `filename` probably -- I don't think basename is useful / helpful (especially if someone is using `--settings-path`)

Reply:
- I added a working test.
- I simplified the string and now use bare filename. 

My thought behind it was to cut the path because we work in the repos context, but with `--settings-path` set I think its better to show the whole path like you said.